src/index.html

```diff
@@ -2,13 +2,29 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta
+      name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+    >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Stop watch</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="stylesheet" href="./styles/index.scss">
   </head>
-  <body>
-    <h1>Stop watch</h1>
+  <body class="page">
+    <main class="main">
+      <div class="watch main__watch">
+        <div class="watch__animation watch__animation--hours">
+          <div class="watch__circle"></div>
+
+          <div class="watch__hours-arrow"></div>
+        </div>
+
+        <div class="watch__animation watch__animation--seconds">
+          <div class="watch__circle"></div>
+
+          <div class="watch__seconds-arrow"></div>
+        </div>
+      </div>
+    </main>
```

Comment:
too complicated code, you can left only one circle in  `watch__animation watch__animation--seconds` element